### ISSUE TITLE
How to add tectosaur2 to cutde

### ISSUE THREAD

**Issue by mehranbasmenji1** [tbenthompson/BIE_book#2]
Hi, 
I am following the TDE examples to run and implement some of "THE QBX SEQUENCE" functions. My Cutde is successfuly installed and the "THE TDE SEQUENCE" section works fine. But when I try to run the examples in the "THE QBX SEQUENCE" section I need the "tectosaur2" module to import some of the functions (e.g. from tectosaur2 import gauss_rule, refine_surfaces, integrate_term, panelize_symbolic_surface). Do you think what is the best way to add it to Cutde? I tried to install "tectosaur2" from the terminal but it seems it is not working properly. I would be happy if you could help me.
I am using Mac.
Cheers,
Mehran

**Comment by tbenthompson** [comment]
Hi, can you detail what you mean when you say "I tried to install "tectosaur2" from the terminal but it seems it is not working properly."

Thanks!

**Comment by mehranbasmenji1** [comment]
Thanks for your reply, I tried to run "python setup.py install" in my Cutde env in conda. I am not sure it is the best way to add "tectosaur2" to Cutde.

**Comment by tbenthompson** [comment]
Hi, I'm going to need *A LOT* more information in order to help you. You will need to describe *exactly* what steps you perform and what the output of each step is.

**Comment by mehranbasmenji1** [comment]
Hi, I installed Cutde and GPU to run some of "the TDE examples in these BIE tutorials" and create some fault models. The examples under "THE TDE SEQUENCE: TRIANGULAR DISLOCATION ELEMENTS" worked fine. 

But, when I started to run examples under "THE QBX SEQUENCE: QUADRATURE BY EXPANSION" there is a module named tectosaur2. For example, I run "from tectosaur2 import gauss_rule" system says "No module named 'tectosaur2". So, I looked at my modules and realized that tectosaur2 is not listed so tried to install it from my terminal (first downloaded the  tectosaur2 and tried to use the "python setup.py install" to install tectosaur2's setup file) to the Cutde environment. 
As a result, Cutde works fine, but I was unable to add  tectosaur2 to my Cutde and use functions that use it (like "from tectosaur2 import XXXX"). 
Thank you for your time.
